src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1228,25 +1271,22 @@ function build_if_list() {
 		'pim' => 'PIM',
 		'ospf' => 'OSPF',
 		'sctp' => 'SCTP',
-		'any' => gettext('any'),
```

Comment:
Points taken - can you take a look at an alternate fix [(8afd8c0)](https://github.com/stilez/pfsense-stilez-working/commit/8afd8c0409a9ed246a244e7a1dbdcb2ad627434b) which seems to meet all requirements neatly, and let me know if it is acceptable for resolving your point?